content/docs/operating/integrations.md

```diff
@@ -59,6 +59,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [prom2teams](https://github.com/idealista/prom2teams): forwards notifications to Microsoft Teams
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [Telegram bot](https://github.com/inCaller/prometheus_bot)
+  * [AWS SNS](https://github.com/DataReply/alertmanager-sns-forwarder)
```

Comment:
Could you move this to the top of the list? It's alphabetically sorted.

Reply:
Sure, done.